### DISCUSSION TITLE
Add pinned scores to profile overlay

### DISCUSSION THREAD

**Discussion by Theighlin** [ppy/osu#16847] [Q&A] [answered]
osu-web:
![image](https://user-images.githubusercontent.com/53081750/153442122-adb493f0-da5c-4e71-af96-9848332546c3.png)
lazer:
![image](https://user-images.githubusercontent.com/53081750/153442080-3c170adf-519f-416b-a79b-da0bb67bc8b1.png)

**Comment by frenzibyte** [comment, marked as answer]
Tracking in #16848.
